agent/base

```diff
@@ -155,36 +155,12 @@ function get_redhat_version() {
     cat /etc/redhat-release | awk '{ print $7 }'
 }
 
-function check_enable_copr {
-    local copr_user=$1
-    local copr_name=$2
-
-    if is_fedora ; then
-        dnf copr enable ${copr_user}/${copr_name}
-    else
-        if is_redhat ; then
-            rhel_version=$(get_redhat_version)
-            rhel_major=${rhel_version%.*}
-            cd /etc/yum.repos.d/
-            local copr_url=https://copr.fedorainfracloud.org
-            local repo_file=${copr_user}-${copr_name}-epel-${rhel_major}.repo
-            [[ ! -f ${repo_file} ]] && wget -c ${copr_url}/coprs/${copr_user}/${copr_name}/repo/epel-${rhel_major}/${repo_file}
-        else
-            echo "Unsupported distribution"
-        fi
-    fi
-}
-
 function check_install_rpm {
     _n="${1}"
     _v="${2}"
-    require-rpm "${_n}" "${_v}"
+    _m="${3}"
+    _rpm="$(require-rpm "${_n}" "${_v}" "${_m}")"
```

Comment:
`require-rpm` needs to emit the rpm with version, if available, on failure, too.

Reply:
Fixed.